applications/nrf5340_audio/src/modules/sd_card.h

```diff
@@ -9,26 +9,28 @@
 
 #include <stddef.h>
 
-/**@brief Print out the contents under SD card root path
+/**@brief Print out the contents under SD card root path and write the content to buffer.
  *
- * @param[in] path	Path of the folder which going to list
+ * @param path	Path of the folder which going to list
  *			If assigned path is null, then listing the contents under root
  *			If assigned path doesn't exist, an error will be returned
- *
+ * @param buf Buffer where data is written. If set to NULL, it will be ignored.
+ * @param buf_size Buffer size
  * @return	0 on success.
  *              -ENODEV SD init failed. SD card likely not inserted
+ *              -EINVAL Failed to append to buffer
  *		Otherwise, error from underlying drivers
  */
-int sd_card_list_files(char *path);
+int sd_card_list_files(char *path, char *buf, size_t buf_size);
 
 /**@brief Write data from buffer into the file
  *
  * @note If the file already exists, data will be appended to the end of the file.
  *
- * @param[in] filename	Name of the target file for writing, the default location is the
+ * @param filename	Name of the target file for writing, the default location is the
```

Comment:
Why has the [in] and [out] been removed? These are mandatory.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/11642#discussion_r1270267041